app/pages/nonsipp/memberdetails/MemberDetailsPage.scala

```diff
@@ -51,26 +51,29 @@ object MembersDetailsPage {
   implicit class MembersDetailsOps(ua: UserAnswers) {
     def membersDetails(srn: Srn): Map[String, NameDOB] = ua.map(MembersDetailsPages(srn))
 
-    def membersOptionList(srn: Srn): List[Option[NameDOB]] = {
```

Comment:
this was the main culprit for the bug - removed usage and created new method. Would have changed it everywhere but the call site code change is too big and the PR is getting heavy